packages/keystone/src/lib/core/mutations/nested-mutation-many-input-resolvers.ts

```diff
@@ -15,33 +15,35 @@ type _UpdateValueType = schema.InferValueFromArg<
 >;
 
 async function getDisconnects(
-  uniqueWheres: (UniqueInputFilter | null)[],
+  uniqueInputs: (UniqueInputFilter | null)[],
   context: KeystoneContext,
   foreignList: InitialisedList
 ): Promise<UniquePrismaFilter[]> {
   return (
     await Promise.all(
-      uniqueWheres.map(async filter => {
-        if (filter === null) return [];
+      uniqueInputs.map(async uniqueInput => {
+        if (uniqueInput === null) return [];
+        const uniqueWhere = await resolveUniqueWhereInput(uniqueInput, foreignList.fields, context);
         try {
-          await context.sudo().db.lists[foreignList.listKey].findOne({ where: filter });
+          await context.sudo().db.lists[foreignList.listKey].findOne({ where: uniqueWhere });
```

Comment:
This isn't right, `context.sudo().db.lists[foreignList.listKey].findOne` should have the values before they go through the unique where input resolvers. (arguably even using `db.lists` is also wrong because the value will be validated and coerced through GraphQL again but it's less wrong since for the types that you would want to use here since they should be able to round-trip as GraphQL inputs (though fixing it would still be good imo))